### DISCUSSION TITLE
[Deploy section] Scheduled deploys

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#42] [Ideas]
Hi, sometimes we need to deploy out of work hours, like during weekends or evening.
It would be a nice feature to schedule the deploys directly from the Mia-Platform console.

Moreover, into the _Deploy history_ section, the user should see the scheduled deploys and, possibly, cancel them.

What do you think about this feature?

**Comment by davidebianchi** [comment]
Hi @silversoul93!

This should absolutely be a feature in our internal developer platform! We should definitely add it to the roadmap!

I would not use the deploy history section but a specific page for scheduled deploy, but yeah this could be seen in the future

**Reply by frengooo** [reply]
Hi @davidebianchi, I just added it to the roadmap.

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
